### PR TITLE
Update CreateAddress

### PR DESCRIPTION
**Pull request by jseagrave21**
**What current issue(s) does this address, or what feature is it adding?**
- Addresses Issue https://github.com/CityOfZion/neo-python/issues/670
- Updates `CreateAddress` functionality
- Improves `CreateAddress` tests

**How did you solve this problem?**
trial and error

**How did you make sure your solution works?**
Unittest

**Are there any special changes in the code that we should be aware of?**
No

**Please check the following, if applicable:**

- [x] Did you add any tests?
- [x] Did you run `make lint`?
- [x] Did you run `make test`?
- [x] Are you making a PR to a feature branch or development rather than master?
- [x] Did you add an entry to `CHANGELOG.rst`? (if not, please do)

### CODE REVIEW

**Comment by jseagrave21** [comment]
Seeing this failure now:

```
FAIL: test_binops (boa_test.tests.test_binops.TestContract)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.3/src/neo-boa/boa_test/tests/test_binops.py", line 50, in test_binops
    tx, results, total_ops, engine = TestBuild(out, ['<<', 16, -2], self.GetWallet1(), '', '07')
AssertionError: ValueError not raised
----------------------------------------------------------------------
Ran 104 tests in 23.470s
FAILED (failures=1)
```

Investigating.

**Comment by jseagrave21** [comment]
@ixje is the the failure I'm seeing above ☝️ something that I need to resolve or it something that you are resolving? The failure looks to come from a test you modified 5 days ago. It is the same test that is failing in  https://github.com/CityOfZion/neo-python/pull/669 but for a different reason.

**Comment by ixje** [comment]
i'm working on resolving that issue...no need to worry at this point

**Comment by jseagrave21** [comment]
Okay thank you @ixje

File: neo/Prompt/Commands/Wallet.py [hunk 1]
```diff
@@ -15,29 +15,30 @@
...
         print('Enter a number greater than 0.')
         return False
-    if int_args > 0 and int_args <= 3:
-        x = int_args + 1
-        while x > 1:
-            wallet.CreateKey()
-            x = x - 1
+
+    address_list = []
+    for i in list(range(int_args)):
```
Comment (ixje) [inline neo/Prompt/Commands/Wallet.py]:
```suggestion
    for i in range(int_args):
```
no need to transform to a list. range already returns an iterator

**Pull request merged**